src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -47,7 +47,7 @@
         "symfony/form": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|^3.0",
+        "symfony/serializer": "~3.2",
```

Comment:
Not sure this PR should be the one triggering the change so adding method_exists checks here would be best, WDYT?